Makefile

```diff
@@ -13,9 +13,10 @@ test_app_instrumentation:
 
 dynamic_properties:
 	$(eval testFilter?=empty)
+	$(eval skipTestsWithPrefix?=)
```

Comment:
I propose to use the same names.
We have multiple options in filters and new entities make harder to understand the logic. I have to map values mentally.

Reply:
Agree. It's because of i cant rename everywhere